src/Altinn.Platform/Altinn.Platform.Storage/Storage/Altinn.Platform.Storage.csproj

```diff
@@ -22,7 +22,7 @@
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.6.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="3.1.13" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.17.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
maybe add a note to the issue so we are extra aware of this change during testing ?

Reply:
Done